net/seafile-server/Makefile

```diff
@@ -8,144 +8,245 @@
-		+libevent2-pthreads +zlib +libzdb +libsqlite3 +libmysqlclient +oniguruma \
-		+libpthread +libuuid +bash +procps-ng +procps-ng-pkill +SEAFILE_FUSE_SUPPORT:libfuse $(ICONV_DEPENDS)
-    MENU:=1
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Seafile server
+  URL:=https://seafile.com/
+  DEPENDS:= \
+    +bash \
+    +glib2 \
+    +jansson \
+    +libarchive \
+    +libevent2 \
+    +libevent2-openssl \
+    +libmariadb \
+    +libopenssl \
+    +libpthread \
+    +libsearpc \
+    +libsqlite3 \
+    +libuuid \
+    +oniguruma \
+    +procps-ng-pkill \
+    +python3-pymysql \
+    +python3-seafile-server \
+    +seafile-ccnet \
+    +seafile-seahub \
+    +sqlite3-cli \
+    +zlib \
+    $(ICONV_DEPENDS)
 endef
 
-define Package/seafile-server/config
-	source "$(SOURCE)/Config.in"
+define Package/seafile-server/description
+  Open source cloud storage with advanced features on privacy
+  protection and teamwork.
 endef
 
-define Package/seafile-server/description
-   Open source cloud storage with advanced features on privacy protection and teamwork.
+define Package/seafile-server-fuse
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Seafile server - FUSE extension
+  URL:=https://seafile.com/
+  DEPENDS:= \
+    +bash \
+    +glib2 \
+    +jansson \
+    +libevent2 \
+    +libevent2-openssl \
+    +libfuse \
+    +libmariadb \
+    +libopenssl \
+    +libpthread \
+    +libsearpc \
+    +libsqlite3 \
+    +libuuid \
+    +procps-ng-pkill \
+    +seafile-ccnet \
+    +seafile-server \
+    +zlib \
+    $(ICONV_DEPENDS)
 endef
 
-CONFIGURE_ARGS += --enable-python
+define Package/seafile-server-fuse/description
+$(call Package/seafile-server/description)
 
-ifeq ($(CONFIG_SEAFILE_FUSE_SUPPORT),y)
-	CONFIGURE_ARGS += --enable-fuse
-	TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include/fuse
-else
-	CONFIGURE_ARGS += --disable-fuse
-endif
+  This package contains the FUSE extension (seaf-fuse) for Seafile
+  server.
+endef
 
-ifeq ($(CONFIG_SEAFILE_CONSOLE_SUPPORT),y)
-	CONFIGURE_ARGS += --enable-console
-else
-	CONFIGURE_ARGS += --disable-console
-endif
+define Package/python3-seafile-server
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Python bindings for Seafile server
+  DEPENDS:=+python3-light +python3-logging +python3-searpc +python3-seafile-ccnet
+  VARIANT:=python3
+endef
 
-PKG_BUILD_DEPENDS:=vala/host libevhtp
+define Package/python3-seafile-server/description
+$(call Package/seafile-server/description)
 
-# This is required as python-package.mk overrides the default setting of having interlinking enabled
+  This package contains Python bindings for Seafile server.
+endef
+
+CONFIGURE_ARGS += \
+	--disable-console \
+	--enable-fuse \
+	--enable-python \
+	--with-mysql="$(STAGING_DIR)/usr/bin/mysql_config"
+
+# This is required as python[3]-package.mk overrides the default setting of having interlinking enabled
 ifdef CONFIG_USE_MIPS16
-	TARGET_CFLAGS += -minterlink-mips16
+  TARGET_CFLAGS += -minterlink-mips16
```

Comment:
Hehehe is this a glib2 workaround?

Reply:
Nvm. This glib2 hack was needed before the package was fixed. The comment suggests something else.